docroot/profiles/custom/cgov_site/modules/custom/cgov_site_section/cgov_site_section.module

```diff
@@ -171,10 +172,18 @@ function cgov_site_section_taxonomy_term_update(EntityInterface $entity) {
   if ($entity->parent->target_id == 0 && $entity->langcode->value === 'en') {
     _cgov_site_section_attempt_set_front_page($entity);
   }
-
   // Save the term children if this term update meets the criteria.
   if (will_trigger_save_children($entity)) {
-    save_term_children($entity);
+    // Checking for any validation related errors.
```

Comment:
Why are you calling validate here? Also, why are you calling validate twice?

Reply:
Also this is not being done in presave. So new terms completely bypass this, or at least should.